easybuild/easyblocks/o/openfoam.py

```diff
@@ -504,27 +510,49 @@ def sanity_check_step(self):
                 else:
                     geom_target_dir = 'triSurface'
```

Comment:
@Neves-P Let's error out in case the `motorBike` tutorial gets moved around again:
```suggestion
                    geom_target_dir = 'triSurface'
            else:
                raise EasyBuildError("motorBike tutorial not found at %s", motorbike_path)